src/screens/market/tezos-info/tezos-info.tsx

```diff
@@ -65,12 +66,12 @@ export const TezosInfo = () => {
         <View style={[styles.commonView, styles.displayCenter]}>
           <View style={styles.percentage}>
             <Text style={styles.subtitle11}>24H</Text>
-            <Text {...getTruncatedProps({ ...styles.subtitle11, color: priceChange24hColor })}>{priceChange24h}</Text>
+            <TruncatedText style={{ ...styles.subtitle11, color: priceChange24hColor }}>{priceChange24h}</TruncatedText>
```

Comment:
I'm not sure that is the difference, but usually we pass styles like this:
`style={[styles.subtitle11, { color: priceChange24hColor }]}`